controllers/UserManagement/TeamEditController.js

```diff
@@ -35,7 +35,7 @@ class TeamEditController extends BaseController {
       req.sessionModel.set('editTeamSuccess', true);
     } catch (err) {
       req.sessionModel.set('editTeamSuccess', false);
-      if (err.status === 409) {
+      if (err.status === 409 || err.status === 403) {
```

Comment:
this if seems unnecessary as 'editTeamAttempt' should always be true when in catch block

Reply:
Attempt and successful attempt are different...it basically sets the logic to say it tried (so a notification appears) and then that it wasn't successful (an error notification). 

The change in the api meant it wasn't returning a 409 for this but a 403, hence expanding the scope - it was failing cause I wasn't an admin but no message. This fixes it...at least I hope 🙂...does locally